foxpuppet/windows/browser/notifications/addons.py

```diff
@@ -33,11 +31,15 @@ def addon_name(self):
         """
         with self.selenium.context(self.selenium.CONTEXT_CHROME):
             if self.window.firefox_version >= 55:
-                return self.root.find_anonymous_element_by_attribute(
-                    'anonid', 'description').find_element(
-                        By.CSS_SELECTOR, '.addon-webext-name').text
+                _addon_name_locator = (
+                    By.CSS_SELECTOR, '#addon-webext-perm-header > \
+                                      .addon-webext-name')
+                return self.root.find_element(*_addon_name_locator).text
             else:
-                label = self.root.find_element(*self._addon_name_locator)
+                _addon_name_locator = (
+                    By.CSS_SELECTOR, '#addon-install-confirmation-content \
+                                      label')
```

Comment:
Can we avoid splitting the string over multiple lines? If this doesn't fit in 80 characters, split between arguments such as:

```python
_addon_name_locator = (
    By.CSS_SELECTOR,
    '#addon-install-confirmation-content label')
```